src/main/java/seedu/duke/data/TaskManager.java

```diff
@@ -28,18 +28,18 @@ public static Task getTask(int taskId) throws TaskNotFoundException {
     /**
      * Edits a task in the Task List.
      *
-     * @param editedTask
-     *  The edited task
      * @param taskId
      *  The index of the task in the Task List.
+     * @param newTaskDescription
+     *  The new description of the task
      * @throws TaskNotFoundException
      *  If
      */
-    public static void edit(Task editedTask, int taskId) throws TaskNotFoundException {
+    public static void edit(int taskId, String newTaskDescription) throws TaskNotFoundException {
```

Comment:
TaskManager's edit function takes a new Task object to get around needing an edit function for each property of task, especially when we will be adding new task properties in the future. Is it possible to reverse this change here?

Reply:
ok